deprecating_process.md

```diff
@@ -1,21 +1,10 @@
 # General Deprecating Process of CRI-O
 
-**What are config options?**
-Config options are the settings that allow us to control various aspects of the behaviour and functionality of CRIO. They allow us to customise how CRI-O operates within a kubernetes cluster.
-
-**Types of config options:**
-
-1)General options: These control overarching settings like logging levels, storage locations, and state directories.
-
-2)Image options: These manage image storage, pulling, and default volumes.
+## What are config options?
 
-3)Network options: These configure container networking, including plugins and configuration directories.
-
-4)Runtime options: These govern the container runtime itself, such as the OCI runtime binary, pause command, and container monitor configuration.
-
-5)Security options: These enforce security measures like AppArmor and seccomp profiles.
+Config options are the settings that allow us to control various aspects of the behaviour and functionality of CRIO. They allow us to customise how CRI-O operates within a kubernetes cluster.

```

Comment:
I'm not a fan of pronouns when writing documents. Something like this is more professional and more apparent, in my opinion.

```suggestion
Config options are the settings that can be used to control various aspects of the behavior and functionality of CRIO. They allow customization of CRI-O operations within a Kubernetes cluster.
```

At the very least, capitalize Kubernetes, and then consider using the American English version of "behavior" vs "behaviour". I'd also suggest changing "config" to "configuration" everywhere.

Reply:
This is my first contribution to open source, still getting the hang of it. I will make it more formal.